File: Dummy/hawkshaw/library/ktextensions/ExceptionsKt.java
package com.hawkshaw.library.ktextensions;

import Y1.K;
import io.sentry.A0;
import io.sentry.C0617v;
import io.sentry.E;

public final class ExceptionsKt {
    public static final void logNonFatal(Exception exc) {
        K.n(exc, "<this>");
        logNonFatal((Throwable) exc);
    }

    public static final void logNonFatal(Throwable th) {
        K.n(th, "<this>");
        th.printStackTrace();
        E b5 = A0.b();
        b5.getClass();
        b5.t(th, new C0617v());
    }
}

File: app/src/main/java/com/hawkshaw/library/ktextensions/ExceptionsKt.kt
package com.hawkshaw.library.ktextensions

import io.sentry.Sentry
import io.sentry.SentryEvent

object ExceptionsKt {
    @JvmStatic
    fun logNonFatal(exc: Exception) {
        logNonFatal(exc as Throwable)
    }

    @JvmStatic
    fun logNonFatal(th: Throwable) {
        th.printStackTrace()
        Sentry.captureEvent(SentryEvent(th))
    }
}